### PR TITLE
added documentation of main interface, updated main interface naming

### PR DESCRIPTION
**Pull request by maciejburda**

### CODE REVIEW

File: example/src/main/java/io/matchmore/sdk/example/MainActivity.kt [hunk 1]
```diff
@@ -60,7 +60,7 @@ class MainActivity : AppCompatActivity() {
     private fun MatchMoreSdk.createPollingSubscription() {
```
Comment (wenjdu) [inline example/src/main/java/io/matchmore/sdk/example/MainActivity.kt]: We should also change the MatchMore to Matchmore.

Let's make a consensus: when we go for SDK we go all caps letter.

MatchmoreSDK and Matchmore

File: examplejava/src/main/java/io/matchmore/sdk/examplejava/MainActivity.java [hunk 2]
```diff
@@ -37,7 +37,7 @@ protected void onCreate(Bundle savedInstanceState) {
         // Creating main device.
         matchMore.startUsingMainDevice(device -> {
```
Comment (wenjdu) [inline examplejava/src/main/java/io/matchmore/sdk/examplejava/MainActivity.java]: Maybe you want to change it in your example as well. 

Let's always go for matchmore with little m for more

File: sdk/src/main/java/io/matchmore/sdk/AlpsManager.kt [hunk 3]
```diff
@@ -65,6 +66,7 @@ class AlpsManager(private val config: MatchMoreConfig) : MatchMoreSdk {
 
     override val locationManager = MatchMoreLocationManager(this)
```
Comment (wenjdu) [inline sdk/src/main/java/io/matchmore/sdk/AlpsManager.kt]: I will let you see where to change for the Matchmore spellings

File: sdk/src/main/java/io/matchmore/sdk/MatchMore.kt [hunk 4]
```diff
@@ -36,48 +36,142 @@ object MatchMore {
 }
 
 interface MatchMoreSdk {
+    /**
+     * Main mobile device created by `startUsingMainDevice()`
```
Comment (wenjdu) [inline sdk/src/main/java/io/matchmore/sdk/MatchMore.kt]: I would go for something more exhaustive like: 
Main device is the device on which the app will be running.
Mobile device object is used to represent the user's smartphone.

Comment (wenjdu) [reply sdk/src/main/java/io/matchmore/sdk/MatchMore.kt]: Let's change in iOS SDK, and then we will sync

**Comment by wenjdu** [comment]
We will follow this document for the naming https://docs.google.com/document/d/19z7QpriFMa6vpBEMp00dkmO3pyYKNS6eQXVJOIzvCZ0/edit

**Comment by maciejburda** [comment]
@wenjdu I took your comments into account.

Please review once again :)

**Pull request merged**